src/hotspot/cpu/riscv/stackChunkFrameStream_riscv.inline.hpp

```diff
@@ -69,8 +69,8 @@ inline intptr_t* StackChunkFrameStream<frame_kind>::unextended_sp_for_interprete
 
 template <ChunkFrames frame_kind>
 intptr_t* StackChunkFrameStream<frame_kind>::next_sp_for_interpreter_frame() const {
-  Unimplemented();
-  return NULL;
+  assert_is_interpreted_and_frame_type_mixed();
+  return (derelativize(frame::interpreter_frame_locals_offset) + 1 >= _end) ? _end : fp() + 2;
```

Comment:
I think "fp() + 2" should be "fp() + sender_sp_offset" instead here.

Reply:
yes; will test it afterward

Reply:
I was wrong before; indeed `fp() + sender_sp_offset` is right.